content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -231,8 +231,7 @@ jobs BEFORE you reference them from another component (via the image-references
 `io.openshift.release.operator` to get automatically included.
 
 1. Ensure you have successfully published your image to the CI integration stream
-1. Follow the [ART instructions](https://mojo.redhat.com/docs/DOC-1179058) to have them build your image
-    * On the [dist-git part of the process](https://mojo.redhat.com/docs/DOC-1168290) it is critical that you ensure your component/image names match as described in the bulleted criteria
+1. Follow the [ART instructions](https://issues.redhat.com/browse/ART-85) to have them build your image.
```

Comment:
I'm pretty sure this is not what we want. There are a _lot_ of instructions on that page (which has moved to https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/guidelines_for_requesting_new_content_managed_by_ocp_art), the ~last of which is to clone https://issues.redhat.com/browse/ART-85

The flow is a little confusing because one of the first instructions on that page directs you here, and then this directs you back at the end. But I don't think it would be improved by skipping ahead to opening the ticket because that would skate over all of the stuff you have to do as a prerequisite (like creating dist-git and comet repos).

Reply:
The ART ticket opens with:

> First read Guidelines for requesting new content managed by OCP ART (at least "Getting Started" section). There is much to be done before you can use this template.

That links to the Source page, and the source page includes:

> Fill out this Jira Request Template to have your build(s) added...

linking back to ART-85.

Since they cross-reference each-other, I don't feel particularly strongly about which gets referenced first.  I am mostly interested in DRYing this up, and while I guess I'm fine if ART wants to dup content between two ART-maintained locations, that's up to them.  My weak preference for linking ART-85 is the nice, compact _Which images (if any) should be shipped as part of OCP?_ table, including the policy about requiring `openshift-e2e-*` and `openshift-e2e-*-serial` workflow tests.  Is ART planning on owning those policies long-term?  In that ART-85 ticket?  Should the "things that need to happen before talking to ART" docs move somewhere that ART doesn't have to maintain them?  Here?  Elsewhere?

Reply:
i'd agree ART-85 is not the best place to express CI requirements; if there's a CI page to link to, i would much prefer that. we don't have any direct interest in CI requirements, we just want the image owner to fulfill them before we spend time productizing.

Reply:
Policy about required tests on OCP repos likely falls under TRT umbrella.

Reply:
Makes sense to me to have TRT set policy here.  But do you agree that this is the place to canonically doc that policy?  There's some precedent for this repo hosting TRT content from #203, although obviously that sort of thing can be refactored if folks think "wait, no, TRT should get their own doc location" or whatever ;)

Reply:
No, I don't think so. This document is CI on-boarding, and I think that OCP-specific content (like this one) should ideally be in at least a separate document, if not somewhere else entirely.